example/vectorAdd/src/main.cpp

```diff
@@ -84,10 +84,18 @@ auto main()
     using Idx = std::size_t;
 
     // Define the accelerator
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
+    using Acc = alpaka::acc::AccGpuCudaRt<Dim, Idx>;
+    using QueueAcc = alpaka::queue::QueueCudaRtSync;
+#elif defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+    using Acc = alpaka::acc::AccGpuHipRt<Dim, Idx>;
+    using QueueAcc = alpaka::queue::QueueHipRtSync;
+#else
     using Acc = alpaka::acc::AccCpuSerial<Dim, Idx>;
+    using QueueAcc = alpaka::queue::QueueCpuSync;
+#endif
```

Comment:
Please remove those things from the examples.

Reply:
This is still valid. The examples are still hardcoded to the serial CPU accelerator.